### PR TITLE
add user documentation for updateUser

### PR DESCRIPTION
**Pull request by DaedalusG**
This is just to add something to the docs about how admins can alter user metadata

### CODE REVIEW

File: doc/admin/how-to/mutate-user-api.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+* This document assumes that you have site-admin level permissions on your Sourcegraph instance
+* Assumes that you have set up your query token [via the steps here](https://docs.sourcegraph.com/api/graphql#quickstart)
+
+## Update a user
+
+First, query the user's ID by using their email address or user name
+
+Example:
+
+```json
```
Comment (unknwon) [inline doc/admin/how-to/mutate-user-api.md:16]: I think `graphql` would work better 😁

Comment (DaedalusG) [reply doc/admin/how-to/mutate-user-api.md:16]: ahh yeah good catch

**Review by northyg — CHANGES_REQUESTED** [review]
Looks good! Just saw a few typos to fix 

> This document (add will here) walk you through the steps of using mutation queries to update or remove users via the GraphQL API consule.

In a couple places Consule -> console.
place holder -> placeholder

File: doc/admin/how-to/mutate-user-api.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# How to update or remove users with the GraphQL API consule
+
+This document walk you through the steps of using mutation queries to update or remove users via the GraphQL API consule.
```
Comment (kevinwojo) [inline doc/admin/how-to/mutate-user-api.md:3]:
```suggestion
This document walk you through the steps of using mutation queries to update or remove users via the GraphQL API console. 
```

File: doc/admin/how-to/mutate-user-api.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
+# How to update or remove users with the GraphQL API consule
```
Comment (kevinwojo) [inline doc/admin/how-to/mutate-user-api.md:1]:
```suggestion
# How to update or remove users with the GraphQL API console
```

**Review by northyg — APPROVED** [review]

**Pull request merged**